src/tokens_entry.rs

```diff
@@ -204,12 +203,6 @@ impl<'a> TokensEntryIter<&'a mut [u8]> {
     }
 
     /// Inserts the given entry data at the right spot.
-    #[pre(
-        "Caller already increased the group size by `size_of::<TOKEN_ENTRY>()`"
```

Comment:
Please make this a "/// Precondition:" comment or something.

Reply:
Done.